src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -309,6 +309,10 @@ std::vector<std::string> disabledTestPatterns() {
         retVector.emplace_back(R"(.*LSTMCellFusion/LSTMCellFusionWithSplitWeights.SubgraphFusedToLSTMCell/(1|8|15))");
         // Ticket: 131541
         retVector.emplace_back(R"(.*smoke_MulticlassNmsLayerTest_dynamic2.*_outType=i32_.*)");
+        // u8 is not supported on arm platform
+        retVector.emplace_back(R"(smoke_LPT.*u8.*)");
```

Comment:
I am not usre it is a good idea to disable all u8 tests. Such limitation is true for ACL ops, but things like Eltwise kernel or KleidiAI works well with unsigned types.

Reply:
`MatMulTransformation` test does not have primitive type in its name and other tests may have not it as well.
I can disable only `MatMulTransformation` u8 tests or remove u8 `MatMulTransformation` test instance at all.